packages/react-dom/server.browser.js

```diff
@@ -9,4 +9,8 @@
 
 'use strict';
 
-module.exports = require('./src/server/ReactDOMServerBrowser');
+var ReactDOMServer = require('./src/server/ReactDOMServerBrowser');
+
+module.exports = ReactDOMServer.default
```

Comment:
Can you please add a similar TODO comment as we have in other such entry points?

Reply:
And it's probably worth mentioning this issue too. In case somebody tries to change it back.